### PR TITLE
docs: cleanup

### PR DESCRIPTION
**Pull request by blueyed**
While messing with Sphinx and tox's docs, this is something that makes sense already.

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -104,7 +104,6 @@ Current features
    config
    support
    changelog
-   links
```
Comment (asottile) [inline doc/index.rst]: kinda funny, you can navigate to this "page" too: http://tox.readthedocs.io/en/latest/links.html

I wonder if we can trick rtfd by underscore prefixing it

Comment (blueyed) [reply doc/index.rst]: We should just remove it, no?

http://tox.readthedocs.io/en/latest/_links.html does not work (404).

Comment (asottile) [reply doc/index.rst]: it's a bit of a _hack_ to DRY links I think? see the `include` lines at the bottom of each file

Comment (asottile) [reply doc/index.rst]: (my suggestion here was to maybe `git mv links.rst _links.rst` and update the includes -- with the thought that maybe rtfd wouldn't serve an html page that's underscore prefixed (entirely a guess, I have no evidence / experience to back this up)

**Pull request merged**